app/models/risk_condition.rb

```diff
@@ -25,8 +25,9 @@ def self.create_condition_for_course_with_type(course_id, type, params, version)
             "Make sure your request body fits the criteria!"
     end
 
-    options = { course_id: course_id, condition_type: type, parameters: params, version: version }
-    new_risk_condition = RiskCondition.new(options)
+    options = { course_id: course_id, condition_type: type, parameters: params.to_hash,
+                version: version }
+    new_risk_condition = RiskCondition.new(options.to_hash)
```

Comment:
See PR description for more details. Essentially, it seems that we can't serialize `ActiveSupport::HashWithIndifferentAccess`. The problem doesn't seem to manifest anywhere else.